packages/react-composites/src/composites/CallComposite/utils/Utils.ts

```diff
@@ -10,6 +10,8 @@ import { isPhoneNumberIdentifier } from '@azure/communication-common';
 const ACCESS_DENIED_TEAMS_MEETING_SUB_CODE = 5854;
 const REMOTE_PSTN_USER_HUNG_UP = 560000;
 const REMOVED_FROM_CALL_SUB_CODES = [5000, 5300, REMOTE_PSTN_USER_HUNG_UP];
+const ROOM_NOT_FOUND_SUB_CODE = 5751;
```

Comment:
do you know if calling will support these as end call reasons is their new end call reason api?

Reply:
Good question. I don't know at this time. I will ask their Teams thread.